### PR TITLE
[Merged by Bors] - Update graffiti.md

### PR DESCRIPTION
**Pull request by WALL-E**
fix typo

### CODE REVIEW

File: book/src/graffiti.md [hunk 1]
```diff
@@ -49,10 +49,12 @@ Below is an example of the validator_definitions.yml with validator specific gra
 ### 3. Using the "--graffiti" flag on the validator client
 Users can specify a common graffiti for all their validators using the `--graffiti` flag on the validator client.
 
+Usage: `lighthouse vc --graffiti emilywest`
```
Comment (michaelsproul) [inline book/src/graffiti.md]:
```suggestion
Usage: `lighthouse vc --graffiti example`
```

What is the significance of `emilywest`?

**Review by michaelsproul — CHANGES_REQUESTED** [review]

File: book/src/graffiti.md (hunk 1 above)
Comment (WALL-E) [reply book/src/graffiti.md]: nothing

**Review by michaelsproul — APPROVED** [review]
Awesome, thanks!

**Pull request closed without merging**